src/modules/loadTasks.js

```diff
@@ -0,0 +1,85 @@
+let tasks = [];
+export const renderTasks = () => {
+  const container = document.getElementById('list');
+  container.replaceChildren();
+  tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  const list = document.createElement('ul');
+  list.setAttribute('class', 'list-group');
+  for (let i = 0; i < tasks.length; i += 1) {
+    const item = document.createElement('li');
+    const span = document.createElement('span');
+    const inputSpan = document.createElement('span');
+    const listLink = document.createElement('a');
+    listLink.style.cursor = 'pointer';
+    listLink.setAttribute('class', 'text-secondary');
+    listLink.style.float = 'right';
+    const listMenu = document.createElement('i');
+    listMenu.setAttribute('class', 'fa fa-ellipsis-v delete');
+    listMenu.setAttribute('id', `delete_${tasks[i].index}`);
+    listLink.appendChild(listMenu);
+    const listContainer = document.createElement('div');
+    const checkbox = document.createElement('input');
+    const editInput = document.createElement('input');
+    editInput.setAttribute('type', 'text');
+    editInput.setAttribute('value', tasks[i].description);
+    editInput.setAttribute('class', 'form-control edit-input');
+    editInput.setAttribute('id', `list_${tasks[i].index}`);
+    // editInput.style.display = "none";
```

Comment:
- [ ] As pointed out by the previous reviewer, please remove all commented code from your codebase